dotCMS/src/main/java/com/dotmarketing/db/DotRunnableThread.java

```diff
@@ -4,19 +4,38 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 import com.dotmarketing.business.DotStateException;
-import com.dotmarketing.exception.DotDataException;
 import com.dotmarketing.portlets.contentlet.model.Contentlet;
 import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
 
 public class DotRunnableThread extends Thread {
 
     final List<DotRunnable> listeners;
+    final List<DotRunnable> flushers;
+    final private Thread networkCacheFlushThread= new Thread("NetworkCacheFlushThread") {
+      @Override
+      public void run() {
 
-    public DotRunnableThread(final List<DotRunnable> listeners) {
+        
+        try {
+          Thread.sleep(Config.getLongProperty("NETWORK_CACHE_FLUSH_DELAY", 3000));
+        } catch (InterruptedException e) {
+          Logger.warn(this.getClass(), e.getMessage());
+        }
+        //Logger.info("networkCacheFlushThread", "starting flushers2:" + flushers.size());
+        flushers.forEach(runner -> runner.run());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'runner' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)